roles/openshift_facts/library/openshift_facts.py

```diff
@@ -393,6 +393,9 @@ def set_metrics_facts_if_unset(facts):
         if 'use_cluster_metrics' not in facts['common']:
             use_cluster_metrics = False
             facts['common']['use_cluster_metrics'] = use_cluster_metrics
+        if 'use_cluster_metrics_hawkular' not in facts['common']:
```

Comment:
This can be set in get_defaults instead of here. The set_<something>_if_unset is an anti-pattern we are trying to get away from.